src/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+
+  --white-color: #fff;
+  --blue-color: #00acdc;
+  --light-grey-color: #616070;
+  --dark-grey-color: #060b35;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: var(--white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__title {
+  margin: 40px 0 0;
+  color: var(--dark-grey-color);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(light-grey-color);
+  margin: 4px 0 0;
+  text-align: left;
+}
+
+.card__rate {
+  margin: 16px 0 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(n) {
+  background-image: url("./images/star-active.svg");
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--dark-grey-color);
+  margin: 0;
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__price {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(light-grey-color);
+  margin: 0;
+  font-weight: 400;
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--dark-grey-color);
+  margin: 0;
+}
+
+.card__buy-link {
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
```

Comment:
you shouldn't hardcode height for elements with text/content. Use paddings instead